src/backend/distributed/utils/maintenanced.c

```diff
@@ -278,10 +281,13 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 		int latchFlags = WL_LATCH_SET | WL_TIMEOUT | WL_POSTMASTER_DEATH;
 		double timeout = 10000.0; /* use this if the deadlock detection is disabled */
 		bool foundDeadlock = false;
+		bool citusHasBeenLoaded = false;
+#ifdef HAVE_LIBCURL
```

Comment:
I guess in this case you're doing more than simple declarations. I still feel there is too much state for the little amount of computation going on here (do we really need all these variables to specify "timeout with a limited number of repetitions"?), but I could be mistaken.